content/developer/accessibility/index.md

```diff
@@ -0,0 +1,254 @@
+
+### Good Mobile Menu (aka. Hamburger Menu)
+
+- Use a real button
+- For "drawer" style (a.k.a. expand and collapse) sections, use a button!
+- Use `aria-expanded`
+- Use `aria-haspopup`
+- You can tab into it after clicking the menu button
+- Close menu on "blur" and on "esc"
+
+### Responsive design
+
+Responsive design is now a requirement for accessibility. 
+
+### Links
+
+Note that by default, not every browser allows tabbing to a link. If a link is part of normal text, that's not an issue, but if it's a part of navigation, or important UI, make sure to put a `tabindex="0"` on it. 
+
+#### Skip Links
+
+Skip links allow users to "skip" over sections and get to the content. The most common type of skip link is to jump over the header. If you're navigating a website using a keyboard,  you probably don't want to be tabbing through the whole header and menu every page load. A common practice is to 
+
+### Tables
+
+- Use `th` for headers 
+- add `scope="col"` or `scope="row"` to define "direction" of heading
+- Add `<caption/>` inside `<table/>` to give an overview, ideally of summary of the contents. A screen reader user can't glace over the table to get an overview.
+
+### Popups/Modals
+
+- `aria-modal="true"`
+- `role="dialogue"`
+- with either `aria-label` or `aria-labelledby`
+- Escape closes popup
+- Set `aria-hidden="true"` on the rest of the page while in the popup
+- tabbing up and down the popup needs to loop you back into the popup. In other words, you shouldn't be able to tab your way out of the popup.
+- When the modal loads, it's got focus on the title (`tabindex="-1"`)
+- When exiting the modal, the focus goes back to what it was before the modal was open
+- If it's a confirmation dialogue, always put focus on least destructive option by default
+
+### Pagination
+- `aria-label` or `aria-labelledby` on container, identifying the contents as pagination
+- `aria-label` or `aria-labelledby` on "next" and "prev"
+- `aria-current="page"` on selected page
+- use links for the "buttons"
+- wrap in `<nav/>` or `role="navigation"`
+
+### Infinite Scrolling
+
+This type of feature is difficult to find information on, since there isn't a consensus on how to handle it yet. However, here are some points to consider:
+
+- Add description of how it works beforehand. This can be hidden from the sighted user using the `sr-only` class.
+- Should be have a way to turn off the infinite scroll.
+- Notify user if new data loads in (politely with a short message using `aria-live="polite"`).
+- Be sure to check swipe gestures with iOS VoiceOver
+- Good examples of accessible infinite scroll is Twitter.
+
+Note that the footer needs to be accessible. With an infinite scroll and a footer, it may be difficult or impossible to reach.
```

Comment:
Maybe a note that a common footer pattern from companies that have implemented this in an accessible way simply don't have a footer (eg. facebook/twitter/linkedin)